src/index.css

```diff
@@ -55,6 +55,7 @@
   --accent-transparent: #fed8231a;
   --secondary-transparent: #ff7bed1a;
   --primary-dark-transparent: #0d00304d;
+  --formal-accent-half-transparent: #efecd680;
```

Comment:
Hm, we do have really close color, may we use it?
![image](https://user-images.githubusercontent.com/49658988/195885953-ce1d5359-30b8-4a48-86ea-c47bae35f7d7.png)

btw, I think we should delete second `accent-light-transparent`

Reply:
fixed, ty